app/src/pages/Register.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+
+const Register = () => {
+  const [username, setUsername] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [repeatPassword, setRepeatPassword] = useState("");
+
+  const handleSubmit = (FormEvent: React.FormEvent) => {
```

Comment:
@majamitic774 Use camelCase for variable names. First latter should not be capitalized.
Instead of `FormEvent` `formEvent`